dss/subscriptions_v2/__init__.py

```diff
@@ -29,10 +29,13 @@ def put_subscription(doc: dict):
 
 
 def get_subscription(replica: Replica, owner: str, uuid: str):
-    item = dynamodb.get_item(table=subscription_db_table.format(replica.name),
-                             hash_key=owner,
-                             sort_key=uuid)
-    return json.loads(item) if item else None
+    try:
+        item = dynamodb.get_item(table=subscription_db_table.format(replica.name),
+                                 hash_key=owner,
+                                 sort_key=uuid)
+        return json.loads(item)
+    except dynamodb.ItemNotFoundInDatabase:
```

Comment:
perhaps logger should be filled with what was missing in this context.

Reply:
This is server side so the user would never see it.